packages/admin-ui-components/src/tooltip/tooltip.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
-import AntTooltip, { AbstractTooltipProps } from "antd/lib/tooltip";
+import { Tooltip as AntTooltip } from "antd";
```

Comment:
All `antd` modules have to be imported as:
```import { Component } from "antd";``` not from `"antd/lib/<component>"`
because `babel-import-plugin` expects `antd` as libraryName (see babel.config.js file)